runtime/doc/syntax.txt

```diff
@@ -4815,6 +4815,9 @@ You can see all the groups currently active with this command: >
     :so $VIMRUNTIME/syntax/hitest.vim
 This will open a new window containing all highlight group names, displayed
 in their own color.
+                                                        *zS-default*
+You can see a short summary of the the highlight groups applied to a given
+position using the keymapping `zS` in normal mode. |default-mappings|
```

Comment:
If Inspect is documented you should reference it IMO. Also this risks duplication with the documentation for inspect.

Reply:
I've reworded it and included the link to `:Inspect`